hair-style.css

```diff
@@ -29,4 +31,94 @@
   display: flex;
   justify-content: space-between;
 
+}
+.body {
+  height: 100%;
+  width: 100%;
+}
+.main-img{
+  max-width: 100%;
+  max-height: 100%;
+}
+.text-box {
+  left: 50%;
+    margin-left: -480px;
+    margin-top: 10%;
+    padding: 0 0 0 470px;
+    position: absolute;
```

Comment:
this is part of what's causing the text to jump out of the .body container. to solve, i'd remove the image from the html and put it in the .body class. and then use flexbox to align the text.